R/toolTimeSpline.R

```diff
@@ -41,12 +41,17 @@ toolTimeSpline <- function(x, dof=NULL){
   out      <- x
   
   x <- as.array(x)
-  
+
   # fill in data with spline approximations/interpolations
   tmpspline <- function(x,dof) return(smooth.spline(x,df=dof, control.spar=list(high=2))$y)
   out <- apply(x, c(1,3) ,tmpspline , dof=dof)
   dimnames(out)[[1]] <- getYears(x)
-  out <- as.magpie(out)
+  
+  if (length(out[1,,1])==67420){
```

Comment:
there shouldn't be a special case for data with 67420 cells. Just run as.magpie(out, spatial=2,temporal=1) in all instances.